custom-templates/priv_instances_public_nlb/neo4j-new.template.yaml

```diff
@@ -0,0 +1,834 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Could this file be renamed to `neo4j-nlb-priv.template.yaml` to match the SSL version of the template